metadata/location/utah_address_points

```diff
@@ -0,0 +1,100 @@
+# Title
+
+Utah Address Points
+
+## ID
+
+4373453d-9e20-44da-9c6e-1f0b1ea653af
+
+## Brief Summary
+
+Contains US Postal Service address points for all twenty-nine counties in Utah.
+
+## Summary
+
+An address point represents a geographic location that has been assigned a US Postal Service (USPS) address by the local address authority (i.e., county or municipality) but does not necessarily receive mail. This data layer contains address points for all twenty-nine counties in Utah.
+
+## Description
+
+### What is the dataset
+
+This dataset contains vector point data for over one million addresses in the State of Utah. Maintaining this dataset is an ongoing effort between UGRC, counties, and local municipalities. UGRC works with each county or municipality's MAL authority to continually improve the address point data.
```

Comment:
I agree with you, I'll go ahead and move it. Beyond specifying that the data is vector point data, is there anything in particular that you think should be added to the "What is the dataset?" section?

Reply:
I think it could be interesting to know how or when a point is created by the county. Like at what phase of a structure or building permit. I'm not sure. You're right thought... what do you out for a description when the summary explains it all?

Reply:
That would be interesting. I did a little research and didn't find too much from USPS about address points, I suppose how they decide to create it likely varies from county to county. I'll ask Zach more about it and see what else we can add.

Reply:
Every county is different. Some will add a point when building begins others add a point when it gets platted. We also don't work with any municipalities, ideally they are feeding their points up to the county.

Reply:
do we know which counties are doing building vs platted or would it be enough to say that it's different and if you're curious contact the county?